hw03_rileyj3.py

```diff
@@ -0,0 +1,111 @@
+######################################################################
+# Author: Bryce Riley
+# Username: rileyj3
+#
+# Assignment: HW03 : Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: To create something complex and learn about github
+# Link to Document: https://docs.google.com/document/d/1EzPApStfIckqskXxnh8CNhMuNgis_GTE2FZZHHZvji4/edit?usp=sharing
+######################################################################
+
+import turtle
+
+wn = turtle.Screen()
+wn.bgcolor('turquoise')
+
+bryce = turtle.Turtle()
+
+
+def build_house():
+    """
+    Have turtle build me a nice house.
+    """
+    pass
+    bryce.speed(0)
+    bryce.pu()
+    bryce.setpos(50, 0)
+    bryce.pd()
+    bryce.color('hot pink')
+    bryce.begin_fill()
+    for i in range(4):
+        bryce.forward(150)
+        bryce.left(90)
+    bryce.end_fill()
+def edit_house():
```

Comment:
Pay attention to the gray squigglies. They indicate style guidelines, such as two blank lines before any function definition.

Reply:
Fixed. Got it!